### PR TITLE
chore: use python3.10 for tests

### PR DESCRIPTION
**Pull request by vchudnov-g**

### CODE REVIEW

**Comment by vchudnov-g** [comment]
https://github.com/googleapis/sphinx-docfx-yaml/pull/294 does not appear to have been enough.

File: tox.ini [hunk 1]
```diff
@@ -12,10 +12,10 @@ commands =
 deps =
     {[testenv]deps}
 commands =
-    python3 -m pytest tests
+    python3.10 -m pytest tests
```
Comment (dandhlee) [inline tox.ini:15]: Instead of specifying them here, we can globally set them like this 

```
[tox]
envlist = 
    tests
    py3.10
```

**Comment by dandhlee** [comment]
Ah, the issue with tests not running is because GH actions is not automatically triggering them, not because of versions issue.

**Comment by vchudnov-g** [comment]
> Ah, the issue with tests not running is because GH actions is not automatically triggering them, not because of versions issue.

So we should undraft https://github.com/googleapis/sphinx-docfx-yaml/pull/340 then, right?

**Comment by dandhlee** [comment]
Yes, just the `ci.yaml` file though - and I remember we had updated guidelines on being able to invoke that recently.

**Review by dandhlee — APPROVED** [review]
Fine to update this file for now. Thank you!

**Pull request merged**